lib/brains/client.py

```diff
@@ -0,0 +1,95 @@
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.http import Http404
+from django_statsd.clients import statsd
+
+import braintree
+from braintree.configuration import Configuration
+from braintree.util.xml_util import XmlUtil
+
+
+class Http(braintree.util.http.Http):
+
+    @staticmethod
+    def raise_exception_from_status(status, message=None):
+        """
+        It would be nice if a 404 from Braintree became a 404 from Django
+        as opposed to a 500 error.
+
+        This method allows us to override the exception response.
+        """
+        statsd.incr('solitude.bango.response.{0}'.format(status))
```

Comment:
also, these are only exceptions which I don't think is what you intended for statsd